security/i2pd/files/0001-i2pd.conf-adjust-for-MacPorts.patch

```diff
@@ -0,0 +1,52 @@
+From c9e23017bcbc2a35712c3e6739ff27176d51c3c6 Mon Sep 17 00:00:00 2001
+From: barracuda156 <vital.had@gmail.com>
+Date: Fri, 11 Nov 2022 07:48:10 +0800
+Subject: [PATCH] i2pd.conf: adjust for MacPorts
+
+---
+ contrib/i2pd.conf | 12 ++++++------
+ 1 file changed, 6 insertions(+), 6 deletions(-)
+
+diff --git a/contrib/i2pd.conf b/contrib/i2pd.conf
+index d0a9e141..06be503e 100644
+--- a/contrib/i2pd.conf
++++ b/contrib/i2pd.conf
+@@ -8,19 +8,19 @@
+ 
+ ## Tunnels config file
+ ## Default: ~/.i2pd/tunnels.conf or /var/lib/i2pd/tunnels.conf
+-# tunconf = /var/lib/i2pd/tunnels.conf
++tunconf = @I2PDDIR@/tunnels.conf
+ 
+ ## Tunnels config files path
+ ## Use that path to store separated tunnels in different config files.
+ ## Default: ~/.i2pd/tunnels.d or /var/lib/i2pd/tunnels.d
+-# tunnelsdir = /var/lib/i2pd/tunnels.d
++tunnelsdir = @I2PDDIR@/tunnels.d
+ 
+ ## Path to certificates used for verifying .su3, families
+ ## Default: ~/.i2pd/certificates or /var/lib/i2pd/certificates
+-# certsdir = /var/lib/i2pd/certificates
++certsdir = @I2PDDIR@/certificates
+ 
+ ## Where to write pidfile (default: i2pd.pid, not used in Windows)
+-# pidfile = /run/i2pd.pid
++pidfile = @PREFIX@/var/run/i2pd.pid
+ 
+ ## Logging configuration section
+ ## By default logs go to stdout with level 'info' and higher
+@@ -30,9 +30,9 @@
+ ##  * stdout - print log entries to stdout
+ ##  * file - log entries to a file
+ ##  * syslog - use syslog, see man 3 syslog
+-# log = file
++log = file
+ ## Path to logfile (default - autodetect)
+-# logfile = /var/log/i2pd/i2pd.log
++logfile = @PREFIX@/var/log/i2pd/i2pd.log
+ ## Log messages above this level (debug, info, *warn, error, none)
+ ## If you set it to none, logging will be disabled
+ # loglevel = warn
```

Comment:
Let remove remove this chunk?

Right now startupitem has log, and redirects all output to the log file. Why not use it instead?